java/server/src/org/openqa/selenium/remote/server/handler/CaptureScreenshot.java

```diff
@@ -17,27 +17,71 @@
 
 package org.openqa.selenium.remote.server.handler;
 
-import static org.openqa.selenium.OutputType.BASE64;
+import static org.openqa.selenium.OutputType.BYTES;
 
 import org.openqa.selenium.TakesScreenshot;
 import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebDriverException;
+import org.openqa.selenium.internal.Base64Encoder;
 import org.openqa.selenium.remote.server.Session;
 
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+
 public class CaptureScreenshot extends WebDriverHandler<String> {
 
+  private File dumpDir;
+
   public CaptureScreenshot(Session session) {
     super(session);
+
+    String path = System.getProperty("dumpScreenshots");
```

Comment:
I couldn't find a way for the `CaptureScreenshot` to get access to the configuration object.